spec/controllers/conferences_controller_spec.rb

```diff
@@ -54,4 +54,16 @@
       end
     end
   end
+
+  describe 'POST create' do
+    context 'unauthenticated users' do
+      let(:params) { { name: 'name', country: 'country', region: 'region', location: 'location', city: 'city', season_id:'1', url: 'www.conference.com' } }
+      it 'cannot create new conference' do
```

Comment:
Could you please leave a blank line before an example block? It's kinda best practise in the Ruby community and in my opinion really increases the readablity.